### ISSUE TITLE
Incompatible with latest release of Peggy.js

### ISSUE THREAD

**Issue by carlossantillana** [metadevpro/ts-pegjs#78]
Peggy.js just released a [breaking release](https://github.com/peggyjs/peggy/releases/tag/v2.0.0) which amongst other things changes how allowedStartRules are defined https://github.com/peggyjs/peggy/pull/175 
As a result, running `node src/generate-parser.js` with the ts-pegjs plugin now causes the error below to occur
```
/path-to-project/node_modules/ts-pegjs/src/passes/generate-ts.js:1076
        options.allowedStartRules.map(
                                  ^

TypeError: Cannot read properties of undefined (reading 'map')
    at generateToplevel (/path-to-project/node_modules/ts-pegjs/src/passes/generate-ts.js:1076:35)
    at generateTS (/path-to-project/node_modules/ts-pegjs/src/passes/generate-ts.js:1562:30)
    at /node_modules/peggy/lib/compiler/index.js:99:9
    at Array.forEach (<anonymous>)
    at /node_modules/peggy/lib/compiler/index.js:96:21
    at Array.forEach (<anonymous>)
    at Object.compile (/node_modules/peggy/lib/compiler/index.js:92:25)
    at Object.generate /node_modules/peggy/lib/peg.js:112:25)
    at /expression-parser/src/generate-parser.js:7:22
    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3)
```

minimal reproducible config

```
// generate-parser.js
var fs = require('fs');
var peggy = require('peggy');
var tspegjs = require('ts-pegjs');

fs.readFile('examples/arithmetics.pegjs', function (err, data) {
  if (err) throw err;
  var parser = peggy.generate(data.toString(), {
    output: 'source',
    cache: true,
    plugins: [tspegjs],
    tspegjs: {
      customHeader: '// @ts-nocheck',
    },
  });
  fs.writeFileSync('examples/arithmetics.ts', parser);
});

```

**Comment by blitz-1306** [comment]
@pjmolina Hey there. Are there any updates on this issues? (:

**Comment by pjmolina** [comment]
- Added support for `peggy 2.0.1` with https://github.com/metadevpro/ts-pegjs/commit/6f9ff31c8a69f463d7c013794fdad3417aa6e492
- Published as `ts-pegjs 2.0.1`

Note that previous version of `ts-pegjs` was supporting peggy `^1.2.0` only.
